docker_files_test.go

```diff
@@ -30,7 +30,7 @@ func TestCopyFileToContainer(t *testing.T) {
 
 	ctr, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
 		ContainerRequest: testcontainers.ContainerRequest{
-			Image: "docker.io/bash",
+			Image: "bash:5.2.26",
```

Comment:
suggestion: we should add constant so its easy to update. We might even consider just using :latest given what it is.